test/MUXControls.Test/Infra/Application.cs

```diff
@@ -116,16 +116,30 @@ internal void Initialize(bool doLaunch = false, string deploymentDir = null)
                 else
                 {
                     Verify.IsTrue(topWindowObj.Matches(_appFrameWindowCondition));
-
                     ApplicationFrameWindow = topWindowObj;
-                    CoreWindow = topWindowObj.Children.Find(_windowCondition);
-                }
+                    UIObject coreWindowObject = null;
+                    bool didFindCoreWindow = false;
+
+                    Log.Comment("Looking for CoreWindow...");
+                    for (int retries = 0; retries < 5; ++retries)
+                    {
+                        if(didFindCoreWindow = topWindowObj.Children.TryFind(_windowCondition, out coreWindowObject))
```

Comment:
nit: 
```suggestion
                        if (didFindCoreWindow = topWindowObj.Children.TryFind(_windowCondition, out coreWindowObject))
```

Reply:
Also consider using the C# syntax to declare out variables instead of needing to forward declare them, like this. Also looks like "didFindCoreWindow" is unused.

```suggestion
                        if (topWindowObj.Children.TryFind(_windowCondition, out var coreWindowObject))
```